TabloidCLI/Repositories/JournalRepository.cs

```diff
@@ -4,10 +4,48 @@
 using TabloidCLI.Models;
 using TabloidCLI.Repositories;
 
-namespace TabloidCLI.Repositories
+namespace TabloidCLI
 {
-    public class JournalRepository : DatabaseConnector
+    public class JournalRepository : DatabaseConnector, IRepository<Journal>
     {
         public JournalRepository(string connectionString) : base(connectionString) { }
+
+
+        public void Insert(Journal journal)
+        {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"INSERT INTO Journal (Title, Content, CreateDateTime)
+                                        OUTPUT INSERTED.Id
+                                        VALUES (@Title, @Content, @CreateDateTime)";
+                    cmd.Parameters.AddWithValue("@Title", journal.Title);
+                    cmd.Parameters.AddWithValue("@Content", journal.Content);
+                    cmd.Parameters.AddWithValue("@CreateDateTime", journal.CreateDateTime);
+                    int id = (int)cmd.ExecuteScalar();
+                    journal.Id = id;
+                }
+            }
+        }
+        public List<Journal> GetAll()
+        {
+            List<Journal> list = new List<Journal>();
+            return list;
+        }
+        public Journal Get(int id)
+        {
+            Journal test = new Journal();
```

Comment:
I don't think a "test" variable is very clear. 

I would rename these to something a little more descriptive  😄 @wes-mitchell

Reply:
You right, I changed those bad boys.